browser/UHR.js

```diff
@@ -180,7 +183,9 @@ function getStatusObject(xhr) {
 		});
 
 	return {
-		code: xhr.status,
+		// handle IE9 bug:
+		// http://stackoverflow.com/questions/10046972/msie-returns-status-code-of-1223-for-ajax-request
+		code: xhr.status === 1223 ? 204: xhr.status,
```

Comment:
I don't get it. The question by link is about IE8 that is completely unsupported by Catberry. What's the point?